tests/ImageSharp.Tests/TestImages.cs

```diff
@@ -61,6 +61,7 @@ public static class Png
         public const string TestPattern31x31 = "Png/testpattern31x31.png";
         public const string TestPattern31x31HalfTransparent = "Png/testpattern31x31-halftransparent.png";
         public const string XmpColorPalette = "Png/xmp-colorpalette.png";
+        public const string AdamHeadsHlg = "Png/adamHeadsHLG.png";
```

Comment:
Looks like you need to use `git rename` to ensure casing. https://github.com/SixLabors/ImageSharp/actions/runs/7032221919/job/19135513822?pr=2592#step:14:53